File: data/java/MAXIMUM_SUM_IARRI_AMONG_ROTATIONS_GIVEN_ARRAY_TRANSLATED.py
====================
def f_gold ( arr , n ) :
    res = np.inf
    for i in range ( n ) :
        curr_sum = 0
        for j in range ( n ) :
            index = ( i + j ) % n
            curr_sum += j * arr [ index ]
        res = max ( res , curr_sum )
    return res

File: data/python/MAXIMUM_SUM_IARRI_AMONG_ROTATIONS_GIVEN_ARRAY_TRANSLATED.java
====================
static int fGold ( int [ ] arr , int n ) {
  int res = - Integer . MAX_VALUE ;
  for ( int i = 0 ;
  i <= n ;
  i ++ ) {
    int currSum = 0 ;
    for ( int j = 0 ;
    j <= n ;
    j ++ ) {
      int index = ( int ) ( ( i + j ) % n ) ;
      currSum += j * arr [ index ] ;
    }
    res = Math . max ( res , currSum ) ;
  }
  return res ;
}

